mesonbuild/backend/xcodebackend.py

```diff
@@ -734,8 +734,9 @@ def generate_xc_build_configuration(self):
                 else:
                     product_name = target.get_basename()
                 ldargs += target.link_args
+                linker, stdlib_args = self.determine_linker_and_stdlib_args(target)
+                ldargs += self.build.get_global_link_args(linker, target.for_machine)
```

Comment:
oh, actually, while you're here... do we hand `self.build.get_project_link_args` in the xcode backend? might as well handle that too...

Reply:
updated